caravel/templates/listings/fullpage.html

```diff
@@ -44,23 +50,30 @@
     </div>
     {% if not is_from_tor() %}
     <div class="col-md-3">
-    {% if current_user and listing.principal.email == current_user.email() %}
+    {% if current_user and listing.principal.email == current_user.email() or is_admin %}
```

Comment:
Right now, yes — since the moderation view doesn't show the sold: status. Eventually, we'd want to get there, but this gets it working for now.